pio/lib/WiFiManagerKT/WiFiManagerKT.cpp

```diff
@@ -1028,7 +1037,7 @@ void WiFiManager::handleMnt()
   page += F("<dd>Date: ");
   page += __DATE__ " " __TIME__;
   page += F("</dd></dl><br>");
-  page += F("<form method='POST' action='/update' enctype='multipart/form-data'><input type='file' name='update'><br><input type='submit' class=\"btn\" value='update'></form>");
+  page += F("<form method='POST' action='/update' enctype='multipart/form-data'><input type='file' name='update' accept='.bin'><br><input type='submit' class=\"btn\" value='update'></form>");
```

Comment:
nice little improvements but somewhat unrelated?

Reply:
Yep.  very unrelated.  I just kept trying to upload the ELF instead of the BIN file... so I change it so it would default to the correct extension.